salt/example.top

```diff
@@ -5,13 +5,5 @@ base:
         - elife.composer
         - elife.nginx
         - elife.nginx-php7
-        - elife.mysql-server
-        - elife.mysql-client
```

Comment:
Could do, but means maintenance in keeping it up to date. Re-add if/when we need it?